bot/cogs/clean.py

```diff
@@ -39,7 +39,8 @@ def mod_log(self) -> ModLog:
     async def _clean_messages(
             self, amount: int, ctx: Context,
             bots_only: bool = False, user: User = None,
-            regex: Optional[str] = None
+            regex: Optional[str] = None,
+            channel: Optional[TextChannel] = None
```

Comment:
This should be dedented one level. I understand though this wasn't something you defined in this PR.